src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const joinSlackUrl =
+  "https://xplorers-io.slack.com/join/shared_invite/zt-27cdb5ouu-taJbzO_Z0E5kbdMQ8nAJ9A";
```

Comment:
Do we still need this Ashis?

Reply:
We don't really need it da. Just left it there, just in case 😄
I'll remove it then